src/replays/_replayswidget.py

```diff
@@ -501,19 +501,24 @@ def onlineTreeClicked(self, item):
                     item.generateInfoPlayersHtml()
 
     def onlineTreeDoubleClicked(self, item):
-        if hasattr(item, "duration"):
+        if hasattr(item, "duration"):  # it's a game not a date separator
             if "playing" in item.duration:  # live game will not be in vault
-                if not item.live_delay:  # live game under 5min
-                    name = None
-                    if item.mod == "ladder1v1":
-                        name = item.name[:item.name.find(" ")]  # "name vs name"
-                    else:
-                        players = (p for t in item.teams for p in t)
-                        for player in players:
-                            name = player["name"]
-                            if name != "" and name in self._playerset:
+                # search result isn't updated automatically - so game status might have changed
+                if item.uid in client.instance.gameset.games:  # game still running
+                    game = client.instance.gameset.games[item.uid]
+                    if not game.launched_at:  # we frown upon those
+                        return
+                    if game.has_live_replay:  # live game over 5min
+                        for name in game.players:  # find a player ...
+                            if name in self._playerset:  # still logged in
+                                self._startReplay(name)
                                 break
-                    self._startReplay(name)
+                else:  # game ended - ask to start replay
+                    if QtWidgets.QMessageBox.question(client.instance, "Live Game ended", "Want to try the Replay?",
```

Comment:
People usually look for games that already ended when they use the replay vault, I think there's no need to ask them anything.

Reply:
yes, but this is the case were the list is still showing 'playing'